.github/workflows/nightly.yaml

```diff
@@ -27,4 +27,4 @@ jobs:
     uses: gardenlinux/gardenlinux/.github/workflows/build.yml@main
     with:
       version: ${{ inputs.version || 'now' }}
-      use_kms: ${{ github.event_name != 'workflow_dispatch' || inputs.use_kms }}
+      use_kms: ${{ inputs.use_kms }}
```

Comment:
`inputs.use_kms` is undefined if not executed as a workflow triggered. To get the result expected you may want to use the value `${{ github.event_name == 'workflow_dispatch' && inputs.use_kms || false }}`

Reply:
will remove this completely since it does not really make sense anyhow